packages/@react-spectrum/dialog/docs/DialogTrigger.mdx

```diff
@@ -0,0 +1,464 @@
+            <Text>
+              This is a popover anchored to the span.
+            </Text>
+          </Content>
+        </Dialog>
+      </DialogTrigger>
+      <span ref={ref} style={{marginInlineStart: '200px', width: '50px'}}>Popover appears over here</span>
+    </div>
+  );
+}
+```
+
+### Placement
+
+The Popover Dialog's placement with respect to its anchor element can be adjusted using the `placement`
+prop. See the props table [below](#props) for a full list of available placement combinations.
+
+```tsx example
+<DialogTrigger type="popover" placement="right top">
+  <ActionButton>Trigger</ActionButton>
+  <Dialog>
+    <Heading>The Heading</Heading>
+    <Divider />
+    <Content>
+      <Text>
+        This is a popover placed to the right of its trigger and offset so the arrow is at the top of the dialog.
+      </Text>
+    </Content>
+  </Dialog>
+</DialogTrigger>
+```
+
+### Offset and Cross Offset
+
+The Popover Dialog's offset with respect to its anchor element can be adjusted using the `offset` and
+`crossOffset` props. `offset` controls the spacing between the Dialog and the anchor element parallel 
+to the direction the Dialog is rendered whereas `crossOffset` handles the offset in the direction perpendicular. 
+
+```tsx example
+<DialogTrigger type="popover" placement="top" offset={-50}>
+  <ActionButton>Trigger</ActionButton>
+  <Dialog>
+    <Heading>The Heading</Heading>
+    <Divider />
+    <Content>
+      <Text>
+        This is a popover offset by an additional 50px above the trigger.
+      </Text>
+    </Content>
+  </Dialog>
+</DialogTrigger>
+
+<DialogTrigger type="popover" placement="top" crossOffset={100}>
+  <ActionButton>Trigger</ActionButton>
+  <Dialog>
+    <Heading>The Heading</Heading>
+    <Divider />
+    <Content>
+      <Text>
+        This is a popover offset by an additional 100px to the right of the trigger.
+      </Text>   
+    </Content>
+  </Dialog>
+</DialogTrigger>
+```
+
+### Flipping
+
+Applying `shouldFlip` to the DialogTrigger makes the Popover Dialog attempt to flip
+on its main axis in situations where the original placement would cause it to render out of view.
+
+```tsx example
+<DialogTrigger type="popover" placement="bottom" shouldFlip>
+  <ActionButton>Trigger</ActionButton>
+  <Dialog>
+    <Heading>The Heading</Heading>
+    <Divider />
+    <Content>
+      <Text>
+        This is a popover that will flip if it can't fully render below the button.
+      </Text>
+    </Content>
+  </Dialog>
+</DialogTrigger>
+```
+
+### Container Padding
+
+You can control the minimum padding required between the Popover Dialog and the 
+surrounding container via the `containerPadding` prop. This affects the positioning
+breakpoints that determine when the Dialog will attempt to flip.
+
+The example below will flip the Dialog from above the trigger button to below the trigger
+button if the Dialog cannot render fully while maintaining 50px of padding between itself and
```

Comment:
This would probably be too much work, but should we calculate when the dialog and 50 pixels below it would be and place a line there to help people see where to position the window to cause a flip? with a button next to it that doesn't fix due to the extra 50px?

Reply:
yeah I think it might be a bit overkill, users can test it by scrolling the screen to different positions and checking what the breakpoint is before the flip occurs